dist/index.js

```diff
@@ -7873,6 +7873,13 @@ function isPullRequestEvent(githubEventName) {
   );
 }
 
+function isForkedPullRequestEvent(githubEventName, githubEventData) {
+  return (
+    isPullRequestEvent(githubEventName) &&
+    githubEventData.pull_request.head.repo.fork
```

Comment:
Do we need any `?.` here to be safe? Or is all of this guaranteed to be there?

Reply:
As far as I can tell from the documentation, it's all guaranteed to be there for pull request and pull request target